### ISSUE TITLE
[LSP] ExpressionAnalyzer::analyze() must be an instance of PhpParser\Node\Expr, null given

### ISSUE THREAD

**Issue by olvlvl** [vimeo/psalm#2188]
Hi everyone,

I'm running Psalm with LSP and it keeps crashing, here are some examples. I'll try to see if I can come up with a fix.

```
PHP Fatal error:  Uncaught TypeError: Argument 2 passed to Psalm\Internal\Analyzer\Statements\ExpressionAnalyzer::analyze() must be an instance of PhpParser\Node\Expr, null given, called in /Volumes/hellofresh/repositories/menus-service/vendor/vimeo/psalm/src/Psalm/Internal/Analyzer/Statements/Expression/CallAnalyzer.php on line 785 and defined in /Volumes/hellofresh/repositories/menus-service/vendor/vimeo/psalm/src/Psalm/Internal/Analyzer/Statements/ExpressionAnalyzer.php:87
Stack trace:
#0 /Volumes/hellofresh/repositories/menus-service/vendor/vimeo/psalm/src/Psalm/Internal/Analyzer/Statements/Expression/CallAnalyzer.php(785): Psalm\Internal\Analyzer\Statements\ExpressionAnalyzer::analyze(Object(Psalm\Internal\Analyzer\StatementsAnalyzer), NULL, Object(Psalm\Context))
#1 /Volumes/hellofresh/repositories/menus-service/vendor/vimeo/psalm/src/Psalm/Internal/Analyzer/Statements/Expression/CallAnalyzer.php(357): Psalm\Internal\Analyzer\Statements\Expression\CallAnalyzer::handleArrayAddition(Object(Psalm\Internal\Analyzer\Statement in /Volumes/hellofresh/repositories/menus-service/vendor/vimeo/psalm/src/Psalm/Internal/Analyzer/Statements/ExpressionAnalyzer.php on line 87
```

```
PHP Notice:  Undefined offset: 0 in /Volumes/hellofresh/repositories/menus-service/vendor/vimeo/psalm/src/Psalm/Internal/Analyzer/Statements/Expression/CallAnalyzer.php on line 780
```

```
PHP Notice:  Trying to get property 'value' of non-object in /Volumes/hellofresh/repositories/menus-service/vendor/vimeo/psalm/src/Psalm/Internal/Analyzer/Statements/Expression/CallAnalyzer.php on line 780
```

**Comment by muglug** [comment]
Thanks, reproduced with
```php
array_push();
```

**Comment by muglug** [comment]
Fixed in 2ffdf1b
